src/index.html

```diff
@@ -3,12 +3,90 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <a href="#home"
+            class="logo__link"
+          >
+            <img src="./images/logo.png"
+              alt="logo"
+              class="logo__img"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list--items">
+              <a href="#"
+                class="link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list--items">
+              <a href="#"
+                class="link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list--items">
+              <a href="#"
+                class="link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__list--items">
+              <a href="#"
+                class="link"
+                data-qa="hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__list--items">
              <a href="#"
                class="link"
              >
                Smartphones
              </a>
            </li>

            <li class="nav__list--items">
              <a href="#"
                class="link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
            </li>
```

![image](https://user-images.githubusercontent.com/80700247/224536490-ae895b7e-3311-431c-893d-c5efc7bcbdef.png)